lisa/schema.py

```diff
@@ -565,6 +565,65 @@ def _call_requirement_method(self, method_name: str, capability: Any) -> Any:
         return value
 
 
+@dataclass_json()
+@dataclass()
+class ACCOptionSettings(FeatureSettings):
```

Comment:
The feature setting should be in the feature file. The Disk and Network is special.